app/styles/app/modules/yaml.scss

```diff
@@ -41,11 +41,19 @@ pre.codedisplay[class*='language-'] {
   }
```

Comment:
```suggestion
    color: $asphalt-grey;
  }
```
The `.codedisplay code` styles below  can be moved up in here

Reply:
done, i think :)

Reply:
Perfect! :smile: Looks good to me, feel free to merge at your convenience :rocket: